app/controllers/products_controller.rb

```diff
@@ -38,6 +38,9 @@ def update
 	end
 
 	def destroy
+		@product.destroy
```

Comment:
Need to do `current_user.shops.find(...).product.find(...).destory`. So we only delete a product that belongs to a shop that the current_user owns. We need this as we don't have an authorisation system yet.